csharp/ql/src/semmle/code/csharp/ir/implementation/raw/internal/TranslatedElement.qll

```diff
@@ -219,32 +223,21 @@ newtype TTranslatedElement =
       // we deal with all the types of initialization separately.
       // First only simple local variable initialization (ie. `int x = 0`)
       exists(LocalVariableDeclAndInitExpr lvInit |
-        lvInit.getInitializer() = expr and
-        not expr instanceof ArrayCreation and
-        not expr instanceof ObjectCreation and
-        not expr instanceof DelegateCreation
+        lvInit.getInitializer() = expr
       )
       or
       // Then treat more complex ones
-      expr instanceof ObjectCreation
-      or
-      expr instanceof DelegateCreation
-      or
       expr instanceof ArrayInitializer
       or
       expr instanceof ObjectInitializer
       or
-      expr = any(ThrowExpr throw).getExpr()
+      expr = any(ThrowStmt throwStmt).getExpr()
```

Comment:
`ThrowExpr` and `ThrowStmt` have the common base class `ThrowElement`; should that be used here instead?

Reply:
That would work, but we do not have `ThrowExpr`s covered yet. I'll modify it thought since there's no harm.